### ISSUE TITLE
`deno test` is noticeably slower on Windows

### ISSUE THREAD

**Issue by dsherret** [denoland/deno#14621]
```shellsession
> git clone https://github.com/dsherret/code-block-writer && cd code-block-writer
> Measure-Command { deno test }
> wsl
$ time deno test
```

Windows: 5.7s
WSL: 2.493s

**Comment by bartlomieju** [comment]
Any chance we could get a flamegraph or other perf profile to see what might be causing it?

**Comment by dsherret** [comment]
Yeah, will do that later. I'm just logging it for now.

**Comment by lucacasonato** [comment]
@dsherret This doesn't happen to be your slow hard drive again, right? :P

**Comment by dsherret** [comment]
Yeah same hard drive. I also bought two M.2 hard drives since then 😄

**Comment by dsherret** [comment]
```
> deno -V
deno 1.33.3
> Measure-Command { deno test -A } # run a few times
11.96s
> deno upgrade
> deno -V
deno 1.33.4
> Measure-Command { deno test -A } # run a few times
2.2s
```

Fixed by #19149 -- thanks @mmastrac!
